nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -403,6 +435,11 @@ Rate Frequency Containment Reserve|Electricity|Gas|OCGT:
    description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
                 for a typical Open Cycle Gas Turbine power plant
    unit: "%"
+   
+Rate Frequency Containment Reserve|Electricity|Gas|CCGT:
+   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
+                for a typical Open Cycle Gas Turbine power plant
```

Comment:
...for a typical Combined Cycle Gas Turbine power plant.

Reply:
thanks